src/rastervision/semseg/data/test/test_batch_norms.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+
+import numpy as np
+from os.path import join
+
+from rastervision.semseg.data.potsdam import (PotsdamNumpyFileGenerator,
+    PotsdamImageFileGenerator)
+from rastervision.common.utils import get_channel_stats
+from rastervision.common.settings import datasets_path
+from rastervision.common.settings import (
+    TRAIN, VALIDATION, TEST
+)
+
+PROCESSED_POTSDAM = 'isprs/processed_potsdam'
+
+class NormalizationTestCase(unittest.TestCase):
+    def test_numpy_potsdam_batch(self):
+        generator = PotsdamNumpyFileGenerator(datasets_path, [0, 1, 2, 3, 4])
+        gen = generator.make_split_generator(
+            TRAIN, target_size=(10, 10), batch_size=100, shuffle=True,
+            augment=False, normalize=True, only_xy=False)
+        batch = next(gen)
+        means, stds = get_channel_stats(batch.all_x)
+
+        print(means, stds)
```

Comment:
You should remove print statements that were just to help debugging before committing.